proposals/NNNN-Dynamic-App-List-Sorting.md

```diff
@@ -0,0 +1,85 @@
+
+As a result, the display order of icons changes every launch. Furthermore, it is very poor visuality and operability (usability) because it takes a long time for the user to find an app that s/he wants to use. In order to solve these problems, we propose to clarify the display order of the SDL app icon and improve the usability by presenting the SDL app icon display sequence and the dynamic app list sorting methods.
+
+## Proposed solution
+
+The following items are proposed to improve the problems mentioned above.
+ 1. Presentation of the dynamic app list sorting methods
+ 2. Clarification of icon display sequence
+
+<b>1. Presentation of the dynamic app list sorting methods</b><br>
+The dynamic app list sorting is performed according to the following procedures.
+
+ 1. Set the priority (such as 1st, 2nd, 3rd ,,,) to the parameter `priority` of the `application_policies` on the policy table. (The OEMs can modify the priorities themselves.)
+ 2. Implemented the `app_list_sorting` parameter below to the SDL Core, sdl_preloaded_pt.json and the policy table. (The OEMs can modify the each parameter value of only the policy table.)
+  - "apphmitype" : The sorting method by using the `apphmitype_sorting_list`.
+  - "alphabet" : The sorting method according to the alphabet order.
+  - "appID" : The sorting method according to the `appID` order of the `application_policies` on the policy table.
+ 3. During the first launch, there is no policy table, therefore it is searched in the default policy table at first, then can be searched in SDL Core to obtain the `app_list_sorting` information. If the 'app_list_sorting' information is not present, the list is sorted how it was previously sorted.
+ 4. The apps are registered to the SDL Core by using the `RegisuterAppInterface`.
+ 5. The sorting order is performed according to the following procedures.
+	1. Prioritize the `priority` parameter of `application_policies` in the policy table.
+	2. Regarding to the apps that there is no priority value in the `priority` parameter, they are prioritized according to the sorting methods which are set by the `sorting_pattern` of the `app_list_sorting` in the policy table.
+ 6. The sorting list is sent to the HMI by using the `UpdateAppList`.
+
+```json
+app_list_sorting: {
+    "sorting_pattern": "apphmitype",
+    "apphmitype_sorting_list": {
+        "NAVIGATION": 1,
+        "MEDIA": 2,
+        "MESSAGE": 3,
+        "INFORMATION": 4,
+        "COMMUNICATION": 5,
+        "SOCIAL": 6,
+        "PROJECTION": 7,
+        "REMOTE_CONTROL": 8,
+        "SYSTEM": 9,
+        "DEFAULT": 10,
+        "BACKGROUND_PROCESS": 11,
+        "TESTING": 12
+    }
+}
+```
+
+By changing the parameters of  `apphmitype_sorting_list` and `priority` created by each OEM in the policy table, they can display the SDL app icon according to each OEM's intended order.
```

Comment:
@theresalech -san,
Thank you for your review.
The word "They" indicates the meaning of  "changing the parameters of  `apphmitype_sorting_list` and `priority`".
Is the sentence "it can display the SDL app icon" correct?